src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -12868,9 +12690,9 @@ LambdaExpressionSyntax parseLambdaExpressionWorker()
                     // Unparenthesized lambda case
                     // x => ...
                     // x!! => ...
-                    var identifier = (this.CurrentToken.Kind != SyntaxKind.IdentifierToken && this.PeekToken(1).Kind == SyntaxKind.EqualsGreaterThanToken) ?
-                        this.EatTokenAsKind(SyntaxKind.IdentifierToken) :
-                        this.ParseIdentifierToken();
+                    var identifier = (this.CurrentToken.Kind != SyntaxKind.IdentifierToken && this.PeekToken(1).Kind == SyntaxKind.EqualsGreaterThanToken)
+                        ? this.EatTokenAsKind(SyntaxKind.IdentifierToken)
```

Comment:
I personally find it strange that we like to put the `?:` tokens at the start of the line, but we like to put the `is`/`or` tokens at the end of the line.

Reply:
Jah.  This is the opinionated IDE style we have.  We have a formatting rule for this as well, but it (like most things) is turned off in teh compiler layer :)